code/clue.js

```diff
@@ -1,59 +1,454 @@
+  const button = (style('revealButton').visibility = 'hidden');
+};
+
+/* Increases the chance of the referred obeject's favourite property to be picked */
+const increaseChance = (object) => {
+  if (object === mystery.killer) {
+    arrayDuplicated = duplicateArray(findFavourite(mystery.killer, weapons));
+    return randomFavourite(weapons, arrayDuplicated);
+  } else if (object === mystery.weapon) {
+    arrayDuplicated = duplicateArray(findFavourite(mystery.weapon, rooms));
+    return randomFavourite(rooms, arrayDuplicated);
+  }
+};
+
+/* The function is called to find the favourite object that matches the id of 
+the favourite property of the mystery object */
+const findFavourite = (mysteryObject, favouriteObjects) => {
+  const favourite = favouriteObjects.find(
+    (favourite) => favourite.id === mysteryObject.favourite
+  );
+  return favourite;
+};
+
+/* The function creates a new array where the favourite object occurs a random
+ number of times */
+const duplicateArray = (favourite) => {
+  const randomNumber = Math.floor(Math.random() * 11);
+  const arrayDuplicated = new Array(randomNumber).fill(favourite);
+  return arrayDuplicated;
+};
+
+/* The functions merges two arrays to create a new array where the favourite object occurs more than once */
+const randomFavourite = (favourite, arrayDuplicated) => {
+  const increasedFavourite = [...favourite, ...arrayDuplicated];
+  return randomSelector(increasedFavourite);
+};
+
+const showInfo = (loaderId) => {
+  if (
+    loaderId === 'loaderWeapon' ||
+    (loaderId === 'loaderRoom' && !killerPicked)
+  ) {
+    get('info').innerHTML = 'Start by picking a killer card';
+  } else if (loaderId === 'loaderRoom' && pickKiller) {
+    get('info').innerHTML = 'Go ahead, pick a weapon card';
+  }
+};
+
+/* Capitalizes the first letter */
+const capitalizeFirstLetter = (inputToCheck) => {
+  return inputToCheck[0].toUpperCase() + inputToCheck.slice(1);
+};
 
-const weapons = []
+/* Function that returns an object to which the id refers */
+const get = (id) => {
+  return document.getElementById(id);
+};
 
-const rooms = []
+/* Function that returns the style property of the referred object */
+const style = (id) => {
+  return get(id).style;
+};
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
-const randomSelector = array => {
-  return array[Math.floor(Math.random() * array.length)]
-}
+/* The fuction is called by the event listener to check which deck is clicked */
+const getLoaderId = (elementId) => {
+  if (elementId === 'loaderKiller') {
+    pickKiller(elementId);
+  } else if (elementId === 'loaderWeapon') {
+    pickWeapon(elementId);
+  } else if (elementId === 'loaderRoom') {
+    pickRoom(elementId);
+  }
+};
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
```

Comment:
remove unnecessary comment to keep the code clean

Reply:
The red lines are already removed from the file. You see the - sign before the code :)